common/constants/constants.go

```diff
@@ -56,12 +56,15 @@ const (
 	DBSTATE_REQUEST_LIM_HIGH = 200
 	DBSTATE_REQUEST_LIM_MED  = 50
 
-	// Replay
+	// Replay -- Dynamic Replay filter based on messages as they are processed.
 	INTERNAL_REPLAY = 1
 	NETWORK_REPLAY  = 2
-	TIME_TEST       = 4  // Checks the time_stamp;  Don't put actual hashes into the map with this.
-	REVEAL_REPLAY   = 8  // Checks for Reveal Entry Replays ... No duplicate Entries within our 4 hours!
-	BLOCK_REPLAY    = 16 // Ensures we don't add the same transaction to multiple blocks.
+	TIME_TEST       = 4 // Checks the time_stamp;  Don't put actual hashes into the map with this.
+	REVEAL_REPLAY   = 8 // Checks for Reveal Entry Replays ... No duplicate Entries within our 4 hours!
+
+	// FReplay -- Block based Replay filter consttructed by processing the blocks, from the database
+	//            then from blocks either passed to a node, or constructed by messages.
+	BLOCK_REPLAY = 1 // Ensures we don't add the same transaction to multiple blocks.
```

Comment:
changing a bitfield bits because block replay is a different variable than the message replay detector